src/iconate.js

```diff
@@ -45,11 +45,24 @@
     }
 
     function changeClasses(element, from, to) {
-        if (element.classList.contains(from)) {
-            element.classList.remove(from);
+        if(Array.isArray(from) && Array.isArray(to)){
```

Comment:
Thanks @TheJoseScream for the PR.
We will need to add these cases 
1. `from` -> `array`, `to` -> `string`
2. `from` -> `string`, `to` -> `array`

when you are there can you also follow the code style (braces/spaces) we follow in the codebase.

Reply:
effectively the method can receive both an array and a string.

Reply:
if you put an array, it will enter the 'if', if not enter the 'else' and it will be taken as a string.

Reply:
Code above only handles the case where from/to are either both array or both are string but not the case where any of them are string and other is array.

Nevermind I will take care of it. once I get some time to work on it.

Reply:
Oh my apologies, now I see, you're right, I'll try to solve it. Thank you!